.github/workflows/ci.yml

```diff
@@ -13,12 +13,16 @@ jobs:
 
     strategy:
       matrix:
-        ruby-version: ["3.0", "2.7", "2.6"]
+        ruby-version:
+          - 3.1
+          - "3.0"
```

Comment:
```suggestion
          - 3.0
```

Reply:
The idea of the quotes is: the YAML float with a zero on it becomes wrongly stringified to "3" and not "3.0", which would mess with interpolation. So, people who care about the output they make with this value, they quote it. There's a GitHub Actions Issue about it.

Reply:
Ah, thanks! Fixing.